epuck_ros2_camera/test/v4l2_to_jpeg.cpp

```diff
@@ -0,0 +1,80 @@
+#include <chrono>
```

Comment:
A small description explaining what the test does would be welcome.

Reply:
All GPU related files are written from scratch